static/js/lierc/commands.js

```diff
@@ -65,6 +65,21 @@ var Commands = function(app) {
     return parts.join(" ");
   });
 
+  add_command("ignores", [], function(panel, line) {
+    var connection = panel.connection;
+    return connection.ignores;
```

Comment:
These should actually not return any value, because whatever they return is sent to the server. I am not sure how best to list the ignores but this won't do it.